core/frontend/src/render/webgl/ContourUniforms.ts

```diff
@@ -0,0 +1,135 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+/** @packageDocumentation
+ * @module WebGL
+ */
+
+import { ColorDef, ContourDisplay, LinePixels } from "@itwin/core-common";
+import { UniformHandle } from "./UniformHandle";
+import { desync, sync } from "./Sync";
+import { Target } from "./Target";
+
+function convertLinePixelsToIndexInShader(linePixels: LinePixels): number {
```

Comment:
The only difference from `lineCodeFromLinePixels` appears to be that it treats an invalid input as "invisible" instead of "solid". What's the reason for that? Under what circumstances do you expect an input that is not a valid `LinePixels` value?

Reply:
I neglected to find that existing function when I wrote this function. I now changed the code to use the existing function (through `LineCode.valueFromLinePixels`). Treating invalid input as solid rather than invisible is fine, and probably preferable.